src/test/librados/list.cc

```diff
@@ -433,9 +438,10 @@ TEST_F(LibRadosList, EnumerateObjects) {
 
   // Ensure a non-power-of-two PG count to avoid only
   // touching the easy path.
-  ASSERT_TRUE(set_pg_num(&s_cluster, pool_name, 11).empty());
-  ASSERT_TRUE(set_pgp_num(&s_cluster, pool_name, 11).empty());
-
+  if (!getenv("CRIMSON_COMPAT")) {
```

Comment:
Let's use the is_crimson_cluster() here -- we only want to reference CRIMSON_COMPAT in the header crimson_env.h header you created.